thrift/src/main/java/com/linecorp/armeria/client/thrift/THttpClient.java

```diff
@@ -19,6 +19,7 @@
 import com.linecorp.armeria.client.ClientBuilderParams;
 import com.linecorp.armeria.client.RpcClient;
 import com.linecorp.armeria.common.RpcResponse;
+import com.linecorp.armeria.common.SessionProtocol;
```

Comment:
I am curious why it was imported without usages.

Reply:
Opps... It is cruft. Because I thought `THttpClient` needs to be changed, but actually not.